src/GeneratedHydrator/CodeGenerator/Visitor/ObjectProperty.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GeneratedHydrator\CodeGenerator\Visitor;
+
+use function version_compare;
+
+/**
+ * @internal
+ * @psalm-immutable
+ */
+final class ObjectProperty
+{
+    /** @var ?string */
+    public $type = null;
+
+    /** @var bool */
+    public $hasDefault = false;
+
+    /** @var ?string */
```

Comment:
This is declared as `string|null`, but is `?bool`

Reply:
`bool` actually, but you're right, wrong copy/paste.